### PR TITLE
Added Stellar spec (CAIP-28)

### PR DESCRIPTION
**Pull request by pitsevich**

### CODE REVIEW

File: CAIPs/caip-28.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+## Abstract
+In CAIP-2 a general blockchain identification scheme is defined. This is the
+implementation of CAIP-2 for the Stellar network.
+
+## Motivation
+See CAIP-2.
+
+## Specification
+
+### Filecoin Namespace
```
Comment (pedrouid) [inline CAIPs/caip-28.md:25]: rename this to "Stellar Namespace"

File: CAIPs/caip-28.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+The namespace "stellar" refers to the wider Stellar ecosystem.
+
+#### Reference Definition
+
+The reference relies on Stellar's current designation of addresses belonging to test or main networks by prefixing them
+with `testnet` or `pubnet` correspondingly.
+
+Reference could only be populated with `testnet` or `pubnet` symbols.
+
```
Comment (pedrouid) [inline CAIPs/caip-28.md:35]: Missing "Resolution Method" section which I'm assuming was also missing on the Filecoin spec 

Refer to [EIP155 namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-3.md) for an example using JSON-RPC and [Cosmos namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-5.md) for an example using REST

File: CAIPs/caip-28.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+# Testnet
+stellar:testnet
+
+# Mainnet
+stellar:pubnet
+```
+
+## Links
+
+- [Filecoin Specification](https://developers.stellar.org/docs)
```
Comment (pedrouid) [inline CAIPs/caip-28.md:58]: rename to "Stellar Specification"

File: CAIPs/caip-28.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+---
+caip: 28
+title: Blockchain Reference for Stellar Namespace
+author: Gleb Pitsevich (@pitsevich)
+discussions-to: TBD
```
Comment (pedrouid) [inline CAIPs/caip-28.md:5]: you can refer this exact PR as a discussions url

**Review by pedrouid — CHANGES_REQUESTED** [review]

File: CAIPs/caip-28.md (hunk 2 above)
Comment (pitsevich) [reply CAIPs/caip-28.md:35]: @pedrouid Thanks for the feedback, added this section.

**Review by pedrouid — APPROVED** [review]
LGTM 👍  @ligi can you give it a last check

File: CAIPs/caip-28.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+discussions-to: https://github.com/ChainAgnostic/CAIPs/pull/44#pullrequestreview-594204708
+status: Draft
+type: Standard
+created: 2021-02-17
+requires: 2
+---
+
+## Simple Summary
+
+This document is about the details of the Stellar network namespace and reference for CAIP-2.
```
Comment (ligi) [inline CAIPs/caip-28.md:14]:
```suggestion
This CAPI defines the Stellar network namespace and reference for CAIP-2.
```

**Review by ligi — CHANGES_REQUESTED** [review]
LGTM - would just change this one wording

**Review by ligi — APPROVED** [review]

File: CAIPs/caip-28.md (hunk 5 above)
Comment (pitsevich) [reply CAIPs/caip-28.md]: @ligi Made the change

Comment (ligi) [reply CAIPs/caip-28.md]: And merged ;-)

**Pull request merged**